examples/docker-compose/README.md

```diff
@@ -31,6 +31,6 @@ In order for you to be able to make edits to the database from the Nhost dashboa
 hasura console
 ```
 
-The Nhost Dashboard uses the [Hasura migrations API](https://hasura.io/docs/latest/hasura-cli/commands/hasura_console/#options) in order to make edits to the database. It runs over port 9693 and is only accessible through running the Hasura console from the CLI. It is not accessible from the Hasura Docker image yet. That is why you need to run it locally. See https://github.com/nhost/nhost/issues/1220.
+The Nhost Dashboard [uses](https://github.com/nhost/nhost/discussions/2398) the [Hasura migrations API](https://hasura.io/docs/latest/hasura-cli/commands/hasura_console/#options) in order to make edits to the database. It runs over port 9693 and is only accessible through running the Hasura console from the CLI. Because Nhost still uses the basic Hasura Docker image that does not include the CLI, that is why you need to run it locally. See https://github.com/nhost/nhost/issues/1220. Users are welcome to contibute a Docker configuration that includes the CLI to resolve this.
```

Comment:
> Because Nhost still uses the basic Hasura Docker image that does not include the CLI

This is not correct either. There is no such image that integrates the cli and graphql-engine together. You need to run them separately. I'd suggest you to look at the docker-compose file we generate with our CLI as that's a complete example and it includes running the hasura cli in a docker container.

For the purposes of this PR I'd suggest sticking to:

```
The Nhost Dashboard uses Hasura's migrations API in order to be able generate migrations. This API endpointis provided by [Hasura's CLI](https://hasura.io/docs/latest/hasura-cli/commands/hasura_console/).
```